engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/repository-details-info.controller.js

```diff
@@ -113,7 +113,7 @@ angular.module('Bastion.repositories').controller('RepositoryDetailsInfoControll
                 repository.os_versions = $scope.osVersionsParam();
                 repository.$update(function (response) {
                     deferred.resolve(response);
-                    $scope.repository.ignore_srpms = $scope.repository.ignorable_content.includes("srpm");
+                    $scope.repository.ignore_srpms = $scope.repository.ignorable_content && $scope.repository.ignorable_content.includes("srpm");
```

Comment:
Was there another bug here? :o

Reply:
Same as above..Console error when $scope.repository.ignorable_content is null..